### ISSUE TITLE
Array toolbar should be visually connected to the array children

### ISSUE THREAD

**Issue by sdirix** [eclipsesource/jsonforms#1524]
**Problem [react/material]**:
 * Array toolbar is not visually connected to its children
 * This is the case for the table as well as the layout renderer

**Examples**:
 * Empty space:
![EmptySpace](https://user-images.githubusercontent.com/8998368/69416932-07d50a80-0d18-11ea-95c7-d8e6367e4110.png)

 * Hard to parse visually that `Analysis` doesn't belong to `Sources`:
![SourcesAnalysis](https://user-images.githubusercontent.com/8998368/69416996-23d8ac00-0d18-11ea-98a1-7e8268a620c1.png)

**Comment by iluuu1994** [comment]
First screenshot: I think it could help if "No data" was wrapped in a paper like the elements.

Second screenshot: Spacing between sources and analysis would probably help. Unrelated but I would also move the errors to the left (like in the upper screenshot).
